src/lib/parquet/src/Flow/Parquet/BinaryReader/Bytes.php

```diff
@@ -47,7 +47,7 @@ public function offsetExists($offset) : bool
 
     public function offsetGet($offset) : mixed
     {
-        return $this->bytes[$offset];
+        return $this->bytes[$offset] ?? null;
```

Comment:
hmm this is a patch, in general, when something is trying to access Bytes out of range it might mean that it's reading not enough values, were you able to reproduce that bug? I would need an example of how to reproduce that issue in order to check what's exactly happening there, because clearly something is wrong and I would rather like to fix it in that place than patch here.

Reply:
I don't have a real sample, but the test generating warning is: `Flow\ETL\Adapter\Parquet\Tests\Integration\ParquetTest::test_writing_with_partitioning` (see #722)

Here are a few caught (open "Test" details in jobs):
- https://github.com/flow-php/flow/actions/runs/6746448114/job/18340393604?pr=720
- https://github.com/flow-php/flow/actions/runs/6746448114/job/18340393369?pr=720
- https://github.com/flow-php/flow/actions/runs/6746448114/job/18340393071?pr=720

Reply:
ok, I'm gonna take a look into this, I think this PR can be closed as this needs to be fixed in a different place that is doing an attempt to read outside the buffer.